test/codegen/array.c

```diff
@@ -11,5 +11,12 @@ int main() {
   a[3] = 2 + a[2];
   __builtin_print(a[3]);
 
+  int b = 2;
+  int c = 0;
+  int d[4] = {1, b, c = 3, 4};
+  for (int i = 0; i < 4; i = i + 1) {
```

Comment:
Suggest not using loop here so that the test doesn't fail when the loop construct is broken.